src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridge.java

```diff
@@ -331,7 +348,7 @@ public CompletionStage<Response> ready() {
      * Retrieves the sink endpoint based on the consumer group and the instance name
      *
      * @param groupId consumer group
-     * @param name instance name
+     * @param name    instance name
```

Comment:
why this change?

Reply:
Ahh its due to my new Mac . I will fix these